common/src/main/java/mca/server/ReaperSpawner.java

```diff
@@ -136,11 +136,23 @@ private boolean isNightTime(World world) {
         return time >= 13000 && time <= 23000;
     }
 
-    private Set<BlockPos> getTotems(World world, BlockPos pos) {
-        return Stream.of(HORIZONTALS).map(d -> pos.offset(d, 3)).filter(pillarCenter ->
-                world.getBlockState(pillarCenter).isOf(Blocks.OBSIDIAN)
-                && world.getBlockState(pillarCenter.down()).isOf(Blocks.OBSIDIAN)
-                && world.getBlockState(pillarCenter.up()).isIn(BlockTags.FIRE)).map(BlockPos::up).collect(Collectors.toSet());
+    private Set<BlockPos> getTotemsFires(World world, BlockPos pos) {
+        int maxPillarHeight = Math.max(2, Config.getInstance().summonGrimReaperAltarMaxPillarHeight); // overridden to 2 if it less than 2.
```

Comment:
Eh, tbf you could make this instead be max between 2 and worldHeight. People wouldn't really want to configure the maximum but moreso the minimum requirements.

So optimally, this line should be Math.max(minPillarHeight, world.topY);